res/skins/Deere/style.qss

```diff
@@ -322,156 +378,90 @@ QTreeView::branch:open:has-children:has-siblings {
   border: 1px solid #1A1A1A;
 }
 /* transition time in Auto DJ tab */
-WLibrary QSpinBox {
+WBaseLibrary QSpinBox {
   min-height: 20px;
   max-height: 20px;
   min-width: 40px;
   max-width: 40px;
 }
 
-WLibrary QSpinBox:editable {
+WBaseLibrary QSpinBox:editable {
   background: transparent;
   color:#d2d2d2;
 }
 
-/* Extra declaration for QRadionButton otherwise it shows up with wrong colors in Linux with Gnome */
-WLibrary QLabel, WLibrary QRadioButton {
-  /* same as QTreeview */
-  color: #d2d2d2;
```

Comment:
This deletion makes the radio button labels in then ``Analyze`` view to display in the default (wrong) color.